city/models.py

```diff
@@ -0,0 +1,10 @@
+from sqlalchemy import Column, Integer, String
+from database import Base
```

Comment:
Ensure that the `Base` class is correctly imported from the `database` module. It should be the declarative base class for your SQLAlchemy models.